### PR TITLE
POC for a better spreadsheet

### PR DESCRIPTION
**Pull request by plaindocs**
For the CFP review meeting, we copy-pasta a spreadsheet that looks a bit like this:

```
Median;[Title](link);Speakers;Tags
4;Talk title; Good people; Tags
3;ANother good one; Folks;
```

While the Median is useful, often you end up with 30 identically scored talks with no secondary sorting. I think it would be more useful to us to have not only Median, but within that Sort by Average. 

This gets all of the talk codes (from which you can build the URL) and titles, as well as all of the scores, associated by talk code.

I ran out of time and motivation to mash the two together, made a separate call for Speakers name info, calculate the average and median, and spit out a sorted spreadsheet.

@mxsasha I don't know if you fancy pushing it over the line? There is no rush, that is for sure. But I can also have another poke when I have a moment.

### CODE REVIEW

**Comment by mxsasha** [comment]
Looks neat, I can pick it up :)

**Comment by plaindocs** [comment]
You know what else would be awesome? 

A spreadsheet of all speakers (including folks who were not accepted), sortable by conference, year or status. 

```
Speaker, Talk, Conference, Year, Accepted status
```

kinda thing

**Comment by plaindocs** [comment]
@mxsasha reckon you'll be able to look at this before the end of the month?

**Review by mxsasha — APPROVED** [review]

File: docs/_scripts/pretalx2cfpsheet.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            'wtd-portland-2020',
+            'write-the-docs-portland-2021',
+            'wtd-portland-2022',
+            'wtd-prague-2020',
+            'write-the-docs-prague-2021',
+            # 'wtd-prague-2022',
+            'write-the-docs-australia-2020',
+            'wtd-australia-and-india-2021',
+        ],
+        output_file='wtd-portland-2022-reviews.csv',
```
Comment (plaindocs) [inline docs/_scripts/pretalx2cfpsheet.py]: Wanna use pretalx_slug here? And add  `*-eviews.csv` to gitignore just in case?

**Review by plaindocs — COMMENTED** [review]
LGTM

**Pull request merged**